docs/Server-configuration.md

```diff
@@ -4,23 +4,23 @@ Some information about all the possible configurations. Click [here](https://git
 
 ## Options
 
-### Server
+### Required Server Parameters
 
 | Key | Default | Description |
 |-|-|-|
-| **PublicIp** | `127.0.0.1` | This needs to the IPv4 address of the server, what you give to others to connect. You can find the your IPv4 address [here](http://whatismyip.host/). Since 1.2.2 it is also possible to use hostnames instead of IPv4 addresses, these must resolve to a valid IPv4 address. |
-| **PublicPort** | `22023` | The public port of the server, what you give to others to  connect. Usually `22023`. |
+| **PublicIp** | `127.0.0.1` | This needs to the IPv4 address of the server, what you give to others to connect. You can find the your IPv4 address [here](http://whatismyip.host/). **Unless you are not exposing this machine to anyone (either privately, in your local network or publicly, in the internet.) You should change this to your public ip. You can still set this to your private ip or "0.0.0.0" (the one you get on `ipconfig`) if you are only exposing this in your local network or you are using a VPS.** Since 1.2.2 it is also possible to use hostnames instead of IPv4 addresses, these must resolve to a valid IPv4 address. |
+| **PublicPort** | `22023` | The public port of the server, what you give to others to connect. (**This is the external port you set within the router when portforwarding.**) Usually `22023`. |
```

Comment:
```suggestion
| **PublicIp** | `127.0.0.1` | This needs to the public IPv4 address of the server which you give to others to connect. You can find your IPv4 address [on this website](http://whatismyip.host/). **Unless you are only planning to use Impostor privately, on your local network, you should change this to your public ip. You can still set this to "0.0.0.0" or your private ip (can be obtained by running `ipconfig` in the command prompt) if you are only using Impostor on your local network, or you are using a VPS.** Is also possible to use hostnames instead of IPv4 addresses, which will be resolved to IPv4 addresses. |
| **PublicPort** | `22023` | The public port of the server which you give to others to connect. (**This is the external port you configure on your router when port forwarding.**) Usually `22023`. |
```